src/electos/ballotmaker/ballots/ballot_layout.py

```diff
@@ -71,31 +74,20 @@
 )
 
 
-def get_election_header() -> dict:
-    return {
-        "Name": "General Election",
-        "StartDate": "2024-11-05",
-        "EndDate": "2024-11-05",
-        "Type": "general",
-        "ElectionScope": "Spacetown Precinct",
-    }
-
-
-def add_header_line(font_size, line_text, new_line=False):
+def add_header_line(
+    font_size: int, line_text: str, new_line: bool = False
+) -> str:
     line_end = "<br />" if new_line else ""
     return f"<font size={font_size}><b>{line_text}</b></font>{line_end}"
 
 
-def build_header_text():
-    elect_dict = get_election_header()
+def build_header_text(election_dict: dict, scope: str) -> str:
```

Comment:
Rename to `election_header_dict`? I found this confusing: I was expecting a full block of election data.

Reply:
Agreed! Updated to `election_header`